elastic/apm-server
# beater: add support for config reloading

## Motivation/summary

Add support for reloading config, to handle Fleet policy updates without restarting the apm-server process.

## Checklist


## How to test these changes

1. Install apm integration package, assign to policy/agent
2. Wait for apm-server to be running and available
3. Update the policy (e.g. enable/disable RUM)
4. Check policy update takes effect without restarting anything

## Related issues

Closes #4376
